bokeh/sampledata/anscombe.py

```diff
@@ -0,0 +1,82 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2021, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+''' A table of `Will Burtin's historical data`_ regarding antibiotic
+efficacies.
```

Comment:
This file is called `anscombe` and the data looks like the Anscombe's quartet dataset. Is it possible something got mixed up here?

Reply:
Yes definitely mixup